modules/features2d/src/brisk.cpp

```diff
@@ -353,13 +353,19 @@ BRISK_Impl::generateKernel(const std::vector<float> &radiusList,
   const int rings = (int)radiusList.size();
   CV_Assert(radiusList.size() != 0 && radiusList.size() == numberList.size());
   points_ = 0; // remember the total number of points
+  double sineThetaLookupTable[n_rot_];
+  double cosThetaLookupTable[n_rot_];
   for (int ring = 0; ring < rings; ring++)
   {
     points_ += numberList[ring];
   }
+  for (size_t rot = 0; rot < n_rot_; ++rot)
```

Comment:
`n_rot_` is hard-codded constant and I propose to calculate the table once or even replace it with pre-computed values.

Reply:
@asmorkalov yes that's pretty much what I did with the trig values - I just don't know if other values in that loop can be precomputed.